docs/interpreter/cassandra.md

```diff
@@ -185,74 +193,96 @@ Insert into users(login,name) vAlues('hsue','Helen SUE');
 ```
 
 ## Syntax Validation
-The interpreters is shipped with a built-in syntax validator. This validator only checks for basic syntax errors.
-All CQL-related syntax validation is delegated directly to **Cassandra**.
 
-Most of the time, syntax errors are due to **missing semi-colons** between statements or **typo errors**.
+The interpreters is shipped with a built-in syntax validator. This validator only checks for basic syntax errors. 
+
+All CQL-related syntax validation is delegated directly to **Cassandra** 
 
+Most of the time, syntax errors are due to **missing semi-colons** between statements or **typo errors**.
+                                    
 ## Schema commands
```

Comment:
I think it's because unnecessarily added spaces next to the `<center>`, `</tr>` tag. 
Here is a screenshot image of mine. 
![screen shot 2016-02-05 at 11 13 50 am](https://cloud.githubusercontent.com/assets/10060731/12836205/f813df58-cbf9-11e5-9932-85ce5c6c06bb.png)

If you delete those spaces, it'll be resolved : )